core-ui/src/main/java/com/maxshop/fragment/BaseBottomSheetFragment.kt

```diff
@@ -18,6 +19,13 @@ open class BaseBottomSheetFragment : BottomSheetDialogFragment() {
         Toast.makeText(requireContext(), text, duration).show()
     }
 
+    open fun addLifecyclerObserver() {}
```

Comment:
Please refactor this logic to interface as we've discussed

Reply:
fixed